packages/arb-token-bridge-ui/src/hooks/useTransactionHistory.ts

```diff
@@ -805,40 +819,57 @@ export const useTransactionHistory = (
     if (page === txPages.length) {
       setPage(prevPage => prevPage + 1)
     }
-  }, [txPages, setPage, page, pauseCount, fetching, runFetcher, isValidating])
-
-  function pause() {
-    setFetching(false)
-  }
+  }, [
+    txPages,
+    setPage,
+    page,
+    pauseCount,
+    fetching,
+    runFetcher,
+    isValidating,
+    pause
+  ])
 
-  function resume() {
-    setFetching(true)
-    setPage(prevPage => prevPage + 1)
-  }
+  const emptyState = useMemo(() => [], [])
+  return useMemo(() => {
+    if (isLoadingTxsWithoutStatus || error) {
+      return {
+        transactions: emptyState,
+        loading: isLoadingTxsWithoutStatus,
+        error,
+        failedChainPairs: emptyState,
```

Comment:
i am not entirely sure if `emptyState` should be reused here

wouldn't both be using the same reference?

i get that we only want to return an empty array for both here and will not change them but this feels very weird?

Reply:
It would, but it should not cause issues as you're not mutating them.